CMakeLists.txt

```diff
@@ -449,6 +455,16 @@ if(NOT ${VELOX_BUILD_MINIMAL})
   include_directories(${Protobuf_INCLUDE_DIRS})
 endif()
 
+if (VELOX_ENABLE_INTEL_IAA)
+  set(inteliaa_SOURCE BUNDLED)
+  resolve_dependency(inteliaa)
+  set(IAA_INCLUDE_DIRS "${inteliaa_SOURCE_DIR}/include"
```

Comment:
Please do not  use globally scoped properties for new dependencies. Linking the relevant targets to `qpl::qpl` is enough. Unsure why you added `PARENT_SCOPE`?